src/main/kotlin/com/arabyte/arabyteapi/domain/article/controller/ArticleController.kt

```diff
@@ -0,0 +1,67 @@
+package com.arabyte.arabyteapi.domain.article.controller
+
+import com.arabyte.arabyteapi.domain.article.dto.article.*
+import com.arabyte.arabyteapi.domain.article.enums.ArticleKind
+import com.arabyte.arabyteapi.domain.article.service.ArticleService
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.data.domain.Sort
+import org.springframework.data.web.PageableDefault
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.*
+
+@Controller
```

Comment:
```suggestion
@RestController
```

restcontroller를 사용하시면 기본적으로 모든 return값에 ResponseEntity.ok()를 감싸줍니다.

Reply:
아 맞네요! 뭔가 안써도 됐던거 같았는데,, 싶었는데 반영했습니다!